aci/data_source_aci_pimextp.go

```diff
@@ -0,0 +1,56 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciExternalProfile() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciExternalProfileRead,
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"l3outside_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"enabled_af": {
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
don't think we need optional in data source schema and we can do computed only for all non-required attributes in data source schema.